src/plugin/regions/region.js

```diff
@@ -149,16 +149,18 @@ export class Region {
      * @param {number} start Optional offset to start playing at
      * */
     playLoop(start) {
-        const s = start || this.start;
-        this.wavesurfer.play(s);
-        this.once('out', () => {
-            const realTime = this.wavesurfer.getCurrentTime();
-            if (realTime >= this.start && realTime <= this.end) {
-                return this.playLoop();
-            }
-        });
+        this.loop = true;
+        this.play(start);
```

Comment:
need to keep the `start || this.start` logic that was in the previous version of the function.

Reply:
`this.play()` is already having it.

Reply:
got it, sorry.